### PR TITLE
Zijie lint fix root files

### PR DESCRIPTION
**Pull request by iTvX**
# Description
lint fix for the files below:
CleanShot 2023-10-26 at 00 39 09@2x

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. **Record a video to ensure all functions are operating normally.**

No need to test the eslint, I have tested, below is the screenshot.

CleanShot 2023-10-26 at 00 41 58@2x

### CODE REVIEW

**Review by emunkh404 — APPROVED** [review]

**Comment by emunkh404** [comment]
I've tested your PR #1460, and I'm pleased to report that there are no issues; all functions are operating normally.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/145945063/401fa1c0-de80-4f99-8edb-1df554fdb943

**Review by DavidC0126 — APPROVED** [review]
Hi @iTvX I tested the PR and it works as intended. 

npxES

Uploading testFunc.mov…

**Comment by beblicarl** [comment]
The app works perfectly. Great work

[screencast-localhost_3000-2023.10.27-12_45_58.webm](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/68056104/2b574686-ad72-4905-b7ce-90bfd1041d5a)

**Review by beblicarl — DISMISSED** [review]

**Review by wantingxu7 — DISMISSED** [review]
Hi @iTvX , I logged as admin and tested the timer, weekly summary, weekly summaries report, user profile. All works well. Good work!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105323719/29ebfd09-de20-4b48-b778-443658efdda4

**Review by Iris-Song — COMMENTED** [review]
I've tested your PR,  all functions are operating normally. But the loading time seems too long.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/20da8bc7-0337-4417-9511-7f0244ec660a

**Review by Shubhu0 — APPROVED** [review]
Hi @iTvX Reviewed and tested your repository! Everything is working as intended, Great work!


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56580102/ca1d2827-5b69-4ba9-bb37-b6794448eb2a

**Review by yaow62023 — DISMISSED** [review]
I have tested your PR, all the functions are work. Good Job! 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/137429365/ff63e48a-9282-4f47-8539-2eadaa602bd2

**Review by yangfan2001 — APPROVED** [review]
Hi, I have tested your PR and everything functions normally. Great Job

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/75376251/e7b2b38b-c792-47cd-b188-1388eeb802fd

**Review by zuhang2000 — APPROVED** [review]
Hi, I've tested the functionality of the APP. Everything is working as expected. All the features are working without a problem. I tested weekly summary, weekly reports, teams, user management, user profile, etc.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/123051008/6c2bde5e-eb6f-4c3d-8349-e6d2494fb17a

**Review by savio-henrique — DISMISSED** [review]
Hi @iTvX , I've tested this PR, everything working as intended, excepting some slow loading of the functions but this is not this PR's fault, as showed in the video below!

Great job! :+1: 

# Video

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/83360232/c0d9b4b8-6cf8-4dba-b001-dee00bbbbad4

File: src/setupTests.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /** Used in jest.config.js */
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (EvianTan) [inline src/setupTests.js:2]: Please fix lint errors in this fix with answers [here](https://stackoverflow.com/questions/44939304/eslint-should-be-listed-in-the-projects-dependencies-not-devdependencies)

**Review by EvianTan — CHANGES_REQUESTED** [review]
Need to actually fix the lint errors in `setupTests.js`.

**Comment by iTvX** [comment]
> Need to actually fix the lint errors in `setupTests.js`.

Hi @EvianTan , this issue has been addressed.

**Review by yi-lin-1234 — COMMENTED** [review]
hello, I've tested your PR, all functions are operating normally. 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92185695/7721ad0d-4f68-4213-a851-ae721cc86c04

**Review by EvianTan — APPROVED** [review]

**Pull request merged**